<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InstaBot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Created by Paris</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="saveBtn">
        <property name="text">
         <string>Save Setting</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runBtn">
        <property name="text">
         <string>Run Bot</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Usename / Email</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="username"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="password"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Time on Each HashTag</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="wait"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Specific Profile</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Number Of Likes on each HashTag</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="numLikes"/>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Spesific Posts URLs</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QTextEdit" name="url"/>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Comment</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QTextEdit" name="comment"/>
      </item>
      <item row="15" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Hash Tags</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <widget class="QTextEdit" name="hashtags"/>
      </item>
      <item row="16" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Like?</string>
        </property>
       </widget>
      </item>
      <item row="16" column="1">
       <widget class="QCheckBox" name="LikecheckBox">
        <property name="text">
         <string>Like Posts</string>
        </property>
       </widget>
      </item>
      <item row="19" column="0">
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Run With Hash Tags</string>
        </property>
       </widget>
      </item>
      <item row="19" column="1">
       <widget class="QCheckBox" name="checkBoxHashTags">
        <property name="text">
         <string>Run With HashTags</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="21" column="0">
       <widget class="QLabel" name="runWithUrlsLabel">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Run With Urls</string>
        </property>
       </widget>
      </item>
      <item row="21" column="1">
       <widget class="QCheckBox" name="runWithUrlsCheckBox">
        <property name="text">
         <string>Run With URLs</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="22" column="0">
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Do comment?</string>
        </property>
       </widget>
      </item>
      <item row="22" column="1">
       <widget class="QCheckBox" name="checkBoxDoComment">
        <property name="text">
         <string>Do comment</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="profile"/>
      </item>
      <item row="20" column="0">
       <widget class="QLabel" name="label_13">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Run With Profile</string>
        </property>
       </widget>
      </item>
      <item row="20" column="1">
       <widget class="QCheckBox" name="checkBoxProfile">
        <property name="text">
         <string>Run With Profile URL</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="msg">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
